canonical-web-and-design/ubuntu.com
# Replace blog pagination with generic component

## Done

Replaced blog pagination with a generic pagination component which can be reused, specifically for `/tutorials`.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: https://ubuntu-com-canonical-web-and-design-pr-6343.run.demo.haus/blog
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that the pagination works
- Check that the pagination matches the [vanilla pattern](https://docs.vanillaframework.io/patterns/pagination/)

## Issue / Card
Fixes #6286

## Screenshots
![blog_pagination_screenshot](https://user-images.githubusercontent.com/501889/72147926-86a10800-3397-11ea-87c5-6e2cec2ca569.png)
